lib/rack/dev-mark/middleware.rb

```diff
@@ -24,7 +24,7 @@ def call(env)
 
         redirect = 300 <= status.to_i && status.to_i < 400
         if !redirect && !Rack::DevMark.tmp_disabled && read_headers['Content-Type'].to_s =~ %r{\btext/html\b}i
-          new_body = ''
+          new_body = String.new('')
```

Comment:
```suggestion
          new_body = +''
```

How about this?

https://docs.ruby-lang.org/ja/latest/method/String/i/=2b=40.html

Reply:
Oh, this's right. Thanks for that :)